rest-doc/examples/SupplierAddress.json

```diff
@@ -1,35 +1,23 @@
 {
-  "id":"3",
-  "salutation":null,
-  "name1":null,
-  "type":"default",
+  "addressId":"central_office",
```

Comment:
Emm.. that's how it is.. or maybe you are confusing `SupplierAddress` with `Supplier`